src/mlpack/core/data/CMakeLists.txt

```diff
@@ -24,9 +24,15 @@ set(SOURCES
   split_data.hpp
   imputer.hpp
   binarize.hpp
+  string_encoding.hpp
+  string_encoding_impl.hpp
   confusion_matrix.hpp
   one_hot_encoding.hpp
   one_hot_encoding_impl.hpp
+  tokenizer/char_split.hpp
```

Comment:
I suggest to change the folder name to its plural form (i.e. `tokenizers`) since other folders are in plural form.

Reply:
Done.